Index: curl/lib/curl_config.h
===================================================================
--- curl.orig/lib/curl_config.h
+++ curl/lib/curl_config.h
@@ -836,9 +836,9 @@
 /* Define absolute filename for winbind's ntlm_auth helper. */
 /* #undef NTLM_WB_FILE */

 /* cpu-machine-OS */
-#define OS "<TARGET_HOST>"
+#define OS "Android"

 /* Name of package */
 #define PACKAGE "curl"

@@ -894,7 +894,7 @@
 #define SELECT_TYPE_ARG5 struct timeval *

 /* Define to the function return type for select. */
-#define SELECT_TYPE_RETV int
+#define SELECT_TYPE_RETV ssize_t

 /* Define to the type qualifier of arg 2 for send. */
 #define SEND_QUAL_ARG2 const

@@ -1068,7 +1068,7 @@
 #endif

 /* Number of bits in a file offset, on hosts where this is settable. */
-/* #undef _FILE_OFFSET_BITS */
+#define _FILE_OFFSET_BITS 64

 /* Define for large files, on AIX-style hosts. */
 /* #undef _LARGE_FILES */
